ext/MRFingerprintingReconCUDAExt/CoilMaps.jl

```diff
@@ -0,0 +1,31 @@
+function MRFingerprintingRecon.calculateCoilMaps(data::CuArray{Complex{T}}, trj::CuArray{T}, img_shape::NTuple{N,Int}; U=CUDA.ones(T, size(trj)[end]), mask=CUDA.ones(Bool, size(trj)[2:end]), kernel_size=ntuple(_ -> 6, N), calib_size=ntuple(i -> nextprod((2, 3, 5), img_shape[i] ÷ (maximum(img_shape) ÷ 32)), length(img_shape)), eigThresh_1=0.01, eigThresh_2=0.9, nmaps=1, verbose=false) where {N,T}
+    @assert all([icalib .== nextprod((2, 3, 5), icalib) for icalib ∈ calib_size]) "calib_size has to be composed of the prime factors 2, 3, and 5 (cf. NonuniformFFTs.jl documentation)."
```

Comment:
Is `U=CUDA.ones(T, size(trj)[end])` a bug? In the CPU-version, the default is an identity matrix, reconstructing each timeframe separately.